### PR TITLE
Refactored & reduced PR

### PR DESCRIPTION
**Pull request by MrMCake**
# Change

- Removed non-testable code
- Expanded properties
- Updated documentation
- Expanded parameter files

## Type of Change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update (Wiki)

### CODE REVIEW

File: arm/Microsoft.ServiceFabric/clusters/.parameters/full.parameters.json [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+                        {
+                            "name": "AppPollIntervalInSeconds",
+                            "value": "60"
+                        }
+                    ]
+                }
+            ]
+        },
+        "managementEndpoint": {
+            "value": "https://sxx-az-sfc-weu-x-001.westeurope.cloudapp.azure.com:19080"
```
Comment (kavishshivhare) [inline arm/Microsoft.ServiceFabric/clusters/.parameters/full.parameters.json:101]: This should be `sxx-az-sfc-full-001`

File: arm/Microsoft.ServiceFabric/clusters/.parameters/full.parameters.json [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "name": {
+            "value": "sxx-az-sfc-full-001"
+        },
+        "tags": {
+            "value": {
+                "resourceType": "Service Fabric",
+                "clusterName": "sxx-az-sfc-x-001"
```
Comment (kavishshivhare) [inline arm/Microsoft.ServiceFabric/clusters/.parameters/full.parameters.json:11]: This should also be `sxx-az-sfc-full-001`.

File: arm/Microsoft.ServiceFabric/clusters/.parameters/full.parameters.json (hunk 1 above)
Comment (MrMCake) [reply arm/Microsoft.ServiceFabric/clusters/.parameters/full.parameters.json]: Fixed

File: arm/Microsoft.ServiceFabric/clusters/.parameters/full.parameters.json (hunk 2 above)
Comment (MrMCake) [reply arm/Microsoft.ServiceFabric/clusters/.parameters/full.parameters.json]: fixed

File: arm/Microsoft.ServiceFabric/clusters/deploy.bicep [hunk 3]
```diff
@@ -30,7 +30,7 @@ param addOnFeatures array = []
 @description('Required. Number of unused versions per application type to keep.')
 param maxUnusedVersionsToKeep int = 3
 
-@description('Optional. Object containing Azure active directory client application ID, cluster application ID and tenant ID.')
+@description('Optional. The settings to enable AAD authentication on the cluster..')
```
Comment (kavishshivhare) [inline arm/Microsoft.ServiceFabric/clusters/deploy.bicep:33]: Please remove the additional full stop (and maybe have the readMe updated too).

**Pull request merged**